test/integration_test.go

```diff
@@ -1236,6 +1236,70 @@ func TestExcludeDirs(t *testing.T) {
 	}
 }
 
+func TestIncludeDirs(t *testing.T) {
+	t.Parallel()
+
+	// Populate module paths.
+	moduleNames := []string{
+		"integration-env/aws/module-aws-a",
+		"integration-env/gce/module-gce-b",
+		"integration-env/gce/module-gce-c",
+		"production-env/aws/module-aws-d",
+		"production-env/gce/module-gce-e",
```

Comment:
If these folders are re-used from other test cases, and this test runs in parallel in those same folders, then I think you're going to have conflicts. You'll want to make a copy of these folders, either now ahead of time, or at run-time using the copy methods you'll find used elsewhere in `integration_test.go`.

Reply:
I went for the copy, we may need to cerate different test cases to better check the include logic,  so I thought it was better to duplicate test/fixture-download/local-with-exclude-dir to test/fixture-download/local-with-include-dir